### DISCUSSION TITLE
关于新增IPV6地址获取形式的建议

### DISCUSSION THREAD

**Discussion by taphep** [jeessy2/ddns-go#274] [Ideas]
非常感谢提供这么好的ddns软件，谢谢！
鉴于IPV6地址的特性，建议增加根据内网终端的mac地址或者DUID地址获取IPV6地址，希望予以考虑，谢谢！

**Comment by jeessy2** [comment]
可以通过正则匹配。其他页面搞复杂了，暂不考虑

**Comment by taphep** [comment]
谢谢回复。
目前获取IPV6地址的方式，是只限于获得安装了软件终端的IPV6地址，建议增加的方式是可以获得其他终端的IPV6地址，比如无法安装ddns-go的终端的IPV6地址。
主要是希望能获得在没有安装DDNS-GO的终端上的IPV6地址，从而绑定域名。

**Reply by jeessy2** [reply]
绝大部分人用不到这个功能，而且实现也有局限性。如果你感兴趣，直接fork自己修改一个版本
